drivers/sensor/lis2dw12/lis2dw12_trigger.c

```diff
@@ -234,6 +234,82 @@ static void lis2dw12_work_cb(struct k_work *work)
 }
 #endif /* CONFIG_LIS2DW12_TRIGGER_GLOBAL_THREAD */
 
+#ifdef CONFIG_LIS2DW12_TAP
+static int lis2dw12_tap_init(const struct device *dev)
+{
+	const struct lis2dw12_device_config *cfg = dev->config;
+	struct lis2dw12_data *lis2dw12 = dev->data;
+
+	LOG_INF("TAP: tap mode is %d", cfg->tap_mode);
+	if (lis2dw12_tap_mode_set(lis2dw12->ctx, cfg->tap_mode) < 0) {
+		LOG_ERR("Failed to select tap trigger mode");
+		return -EIO;
+	}
+
+	LOG_INF("TAP: ths_x is %02x", cfg->tap_threshold[0]);
+	if (lis2dw12_tap_threshold_x_set(lis2dw12->ctx, cfg->tap_threshold[0]) < 0) {
+		LOG_ERR("Failed to set tap X axis threshold");
+		return -EIO;
+	}
+
+	LOG_INF("TAP: ths_y is %02x", cfg->tap_threshold[1]);
+	if (lis2dw12_tap_threshold_y_set(lis2dw12->ctx, cfg->tap_threshold[1]) < 0) {
+		LOG_ERR("Failed to set tap Y axis threshold");
+		return -EIO;
+	}
+
+	LOG_INF("TAP: ths_z is %02x", cfg->tap_threshold[2]);
+	if (lis2dw12_tap_threshold_z_set(lis2dw12->ctx, cfg->tap_threshold[2]) < 0) {
+		LOG_ERR("Failed to set tap Z axis threshold");
+		return -EIO;
+	}
+
+	if (cfg->tap_threshold[0] > 0) {
+		LOG_INF("TAP: tap_x enabled");
+		if (lis2dw12_tap_detection_on_x_set(lis2dw12->ctx, 1) < 0) {
+			LOG_ERR("Failed to set tap detection on X axis");
+			return -EIO;
+		}
+	}
+
+	if (cfg->tap_threshold[1] > 0) {
+		LOG_INF("TAP: tap_y enabled");
+		if (lis2dw12_tap_detection_on_y_set(lis2dw12->ctx, 1) < 0) {
+			LOG_ERR("Failed to set tap detection on Y axis");
+			return -EIO;
+		}
+	}
+
+	if (cfg->tap_threshold[2] > 0) {
+		LOG_INF("TAP: tap_z enabled");
+		if (lis2dw12_tap_detection_on_z_set(lis2dw12->ctx, 1) < 0) {
+			LOG_ERR("Failed to set tap detection on Z axis");
+			return -EIO;
+		}
+	}
+
+	LOG_INF("TAP: shock is %02x", cfg->tap_shock);
+	if (lis2dw12_tap_shock_set(lis2dw12->ctx, cfg->tap_shock) < 0) {
+		LOG_ERR("Failed to set tap shock duration");
+		return -EIO;
+	}
+
+	LOG_INF("TAP: latency is %02x", cfg->tap_latency);
+	if (lis2dw12_tap_dur_set(lis2dw12->ctx, cfg->tap_latency) < 0) {
+		LOG_ERR("Failed to set tap latency");
+		return -EIO;
+	}
+
+	LOG_INF("TAP: quiet time is %02x", cfg->tap_quiet);
```

Comment:
`LOG_DBG`?

Reply:
You mean all the LOG_INF?

Reply:
My feeling is that most LOG_INF here should be LOG_DBG

Reply:
> My feeling is that most LOG_INF here should be LOG_DBG

Done for all

Reply:
@gmarull is that fine now?